components/definitions/playlist.d.ts

```diff
@@ -0,0 +1,9 @@
+declare namespace Playlist {
```

Comment:
We do not need a global definition type for this since this file is not describing an API or anything global.

It would be better to keep this local to `components/playlist/resources`, e.g. as `PlaylistState.ts` which would look something like:

```ts
export type PlaylistState = {
  lists: []
}
```

(and also the `[]` needs to actually define what type will be included in the array, but that's a separate issue!)